web_external/vue/views/AnnotationTool.vue

```diff
@@ -0,0 +1,280 @@
+<template lang="pug">
+.annotation-tool-container.container-fluid
+  .isic-page.row
+    .isic-pane.col-sm-3
+      .header
+        .info-group
```

Comment:
This group is not necessary.

Reply:
Removing these grounds  changes the style--specifically padding--compared to the original implementation.